Userland/Applications/Presenter/Presentation.cpp

```diff
@@ -129,20 +130,33 @@ ErrorOr<NonnullOwnPtr<Presentation>> Presentation::load_from_file(StringView fil
     if (maybe_metadata.is_null() || !maybe_metadata.is_object() || maybe_slides.is_null() || !maybe_slides.is_array())
         return Error::from_string_view("Metadata or slides in incorrect format"sv);
 
+    auto const& maybe_templates = global_object.get("templates"sv);
+    if (!maybe_templates.is_null() && !maybe_templates.is_object())
```

Comment:
Pretty sure this should be an or, not an and:
```suggestion
    if (!maybe_templates.is_null() || !maybe_templates.is_object())
```

Reply:
🤦 thx for catching this bug early